cmd/licensecheck/main.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Command licensecheck provides a command-line interface to the licensecheck
+// package. Given a file, it prints any licenses found in it, one per line,
+// along with the percentage of the license text that matched. It exits with a
+// non-zero status code on error; finding no licenses is not considered an
+// error.
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/google/licensecheck"
+)
+
+func main() {
+	if len(os.Args) != 2 {
```

Comment:
please use the flag package and add a flag like -v that prints all the fields of the Match structs.

Reply:
Good call. Done.